app/Formatter/BoardSwimlaneFormatter.php

```diff
@@ -104,19 +105,45 @@ public function format()
             }
         }
 
+        $this->addNumberOfOpenTasksInColumn();
+
         return $this->swimlanes;
     }
 
     /**
-     * Calculate stats for each column acrosss all swimlanes
+     * Add number of open tasks to eeach column, ignoring the current filter
+     *
+     * @access protected
+     */
+    protected function addNumberOfOpenTasksInColumn()
+    {
+        $column_open_task = $this->db->table(TaskModel::TABLE)
```

Comment:
This is better but we should avoid doing SQL queries in a Formatter class. This class is about formatting the results from the database.

That would be nice if you could find a way to move this method to the models.

Reply:
@fguillot I've changed it again. It now uses an existing method which returns to taskcount.